PendingReleaseNotes

```diff
@@ -9,6 +9,18 @@
   that were storing state in RADOS omap, especially without striping which
   limits scalability.
 
+* MGR: pg_autoscaler module has the option to switch to a `scale-down` profile
```

Comment:
This is a good addition for the docs - should add this to the "Automated scaling" section of doc/rados/operations/placement-groups.rst 

For the release notes I'd suggest something a bit shorter, e.g.

The pg_autoscaler has a new 'scale-down' profile which provides more performance from the start for new pools. The default remains the existing behavior, now called the 'scale-up' profile, since changing profiles causes data movement. For more details, see <link to docs>